c3po/job/utils/metadata.py

```diff
@@ -0,0 +1,96 @@
+from c3po.db.dao import user, link, song, artist, genre
```

Comment:
Nitpick: Can you import the classes within these modules into the [`__init__.py`](https://github.com/lttkgp/C-3PO/blob/bb208c5cccda4c4de4a7ad84195b9b3517d9ad9b/c3po/db/dao/__init__.py) in `c3po.db.dao` package? Example: from user import User.

That way, we can change this line to read as `from c3po.db.dao import User, Link, Song, Artist, Genre`. Then everywhere in the code in this module, we can directly use, say, `User` rather than `user.User`. Just makes the code cleaner when working with modules with nothing except a class inside.